tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -479,7 +479,17 @@ describe('Test Self-Healing on IMMUTABLE resources when correctDrift is enabled'
           // Adding more wait for 30seconds to capture the error if occurred after modifying the resources.
           cy.wait(30000);
           cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
-          cy.verifyTableRow(0, 'Active', repoName);
+
+          // As per comment in the issue: https://github.com/rancher/fleet/issues/2609#issuecomment-2504051729
```

Comment:
The issue occurs in 2.10 as stated in the comment. Yet, the test passes there.
I don't know why this would work around it.  Did the branch test pass?

Reply:
I know it did not pass and as commented it did on 2.10; that is why it seems unrelated the reason given unless I am missing something else. Feel free to skip it, but I am not convinced is for that reason.

Reply:
As discussed, I am skipping the test.